src/main/views/users/tabs/inviteUserContent.njk

```diff
@@ -0,0 +1,133 @@
+{% from "govuk/components/tabs/macro.njk" import govukTabs %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/fieldset/macro.njk" import govukFieldset %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+<h2 class="govuk-heading-l">Create A User</h2>
+
+{% if errors|length %}
+  {{ govukErrorSummary({
+    titleText: "There is a problem",
+    errorList: errors
+  }) }}
+{% endif %}
+
+
+{% set emailsError = false %}
+  {% if account.email === '' %}
+    {% set emailsError = { text: 'Email address is required' } %}
+  {% elif account.isInvalidFormat === true %}
+    {% set emailsError = { text: 'Invalid email address format' } %}
+  {% endif %}
+
+{% set firstNameError = false %}
+  {% if account.firstName === '' %}
+   {% set firstNameError = { text: 'First Name is required' } %}
+  {% endif %}
+
+ {% set lastNameError = false %}
+   {% if account.lastName === '' %}
+    {% set lastNameError = { text: 'Last Name is required' } %}
+  {% endif %}
+
+ {% set rolesError = false %}
+   {% if not account.roles|length and errors|length  %}
+   {% set rolesError = { text: 'Assigned role is required' } %}
+ {% endif %}
+
+ {% set adminRoleChecked = false %}
+ {%  if account.roles|length and account.roles[0] === 'fact-admin'  %}
+  {% set adminRoleChecked = true %}
+ {% endif %}
+
+ {% set superAdminRoleChecked = false %}
+ {%  if account.roles|length and account.roles[0] === 'fact-super-admin'  %}
+   {% set superAdminRoleChecked = true %}
+  {% endif %}
+
+
+{% call govukFieldset({
+  }) %}
+
+  {{ govukInput({
+    label: {
+      text: "Email"
+    },
+    classes: "govuk-input--width-20",
+    id: "email",
+    name: "account[email]",
+    value: account.email,
+    errorMessage: emailsError
+  }) }}
+
+  {{ govukInput({
+    label: {
+      text: "First Name"
+    },
+    classes: "govuk-input--width-20",
+    id: "firstName",
+    name: "account[firstName]",
+    value: account.firstName,
+    errorMessage: firstNameError
+  }) }}
+
+{{ govukInput({
+    label: {
+      text: "Last Name"
+    },
+    classes: "govuk-input--width-20",
+    id: "lastName",
+    name: "account[lastName]",
+    value: account.lastName,
+    errorMessage: lastNameError
+  }) }}
+
+
+{{ govukRadios({
+  classes: "govuk-radios--small",
+  id: "roles",
+  name: "account[roles][]",
```

Comment:
account-role may be better than account[roles][], but only a small one really. Base the change on how much efforts required